neonvm/client/informers/externalversions/neonvm/v1/ippool.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2022.
```

Comment:
Just passing by, but noticed that there are big chunks copied from other repos, like this
https://github.com/kubesphere/kubesphere/blob/c12820019206b04b3f3c3753b136afa89aaf7f73/pkg/simple/client/network/ippool/calico/client/informers/externalversions/network/calicov3/ippool.go#L2
So I have two questions

1. Can we not copy-paste them, but build our code on top of them? As far as I see, it's not an internal package in `kubesphere`, so should be exported. At the same time in this code we retrun `neonvm1`-based types, but maybe it's possible to do with a tiny wrapper around `kubesphere` interfaces?

2. If there is no way except copy-pasting, maybe let's at least reference original packages? So in case we will have any problems with this code, we can look for updated versions. This `kubesphere` seems to be actively developed

Reply:
`neonvm/client/*` is generated by client generator, I will try to look how it goes.  Calico libs not used in neonvm, but calico client generated by the same generator I think